File: solutions/Mangai94/Triangle.cs
using System;

namespace sample{

    class Triangle : Shape
    {
        public Triangle(double height, double width): base(height, width)
        {
            Console.WriteLine("Triangle constrcutor called");
        }

        
        public override double compute_area()
        {
            
            return (this.height * this.width) / 2;
        }
    }
}

File: solutions/laxminarayankatker/src/solutions/laxminarayankatker/Triangle.java
package solutions.laxminarayankatker;

public class Triangle  extends Shape {

    public Triangle() {
        super();
    }
    public Triangle(double height,double width) {
        super(height, width);
    }
    public double compute_area()
    {
        return 0.5*(this.height*this.width);
    }

}
